glide.yaml

```diff
@@ -1,39 +1,56 @@
 package: github.com/bblfsh/server
 import:
+- package: github.com/Sirupsen/logrus
+  version: ^1.0.2
 - package: github.com/bblfsh/sdk
+  version: 9db6b2ee9500857b54f8d40165dc92aba482aadf
```

Comment:
but a `glide up` wouldn't update this file, so we have to track specific commits by hand, is that correct?